test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs

```diff
@@ -63,16 +63,16 @@ public void CommandLineRunSettingsShouldWinAmongAllOptions(RunnerInfo runnerInfo
                         string.Concat("RunConfiguration.TestAdaptersPaths=" , this.GetTestAdapterPath())
                     });
 
-            this.RunTestWithRunSettings(runConfigurationDictionary, runSettingsArgs, additionalArgs, testhostProcessName, expectedNumOfProcessCreated);
+            await this.RunTestWithRunSettings(runConfigurationDictionary, runSettingsArgs, additionalArgs, testhostProcessName, expectedNumOfProcessCreated);
         }
 
         /// <summary>
-        /// Command line run settings should have high precedence btween cli runsettings and cli switches.
+        /// Command line run settings should have high precedence between cli runsettings and cli switches.
```

Comment:
Let me know if you prefer this typo fix to be in a separate PR.

Reply:
It's fine to include somewhere. You don't have to start a new PR for each change :)